### DISCUSSION TITLE
Interpreting variable contents as typst code

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#2285] [Q&A] [answered]
Consider the following `typst` document:

```
#let x = "*word*"

#x
```

The PDF output is 

`*word*`

but I would like it to be `word` in bold.

How can we accomplish that? Thanks!

**Comment by Enter-tainer** [comment, marked as answer]
#eval https://typst.app/docs/reference/foundations/eval/

**Reply by psads-git** [reply]
Thanks, @Enter-tainer: that is it!
